bin/src/run/build.ts

```diff
@@ -10,7 +10,7 @@ import { BatchWarnings } from './batchWarnings';
 import { SourceMap } from 'magic-string';
 
 export default function build (inputOptions: InputOptions, outputOptions: OutputOptions[], warnings: BatchWarnings, silent = false) {
-	const useStdout = outputOptions.length === 1 && !outputOptions[0].file && inputOptions.input instanceof Array === false;
+	const useStdout = outputOptions.length === 1 && !outputOptions[0].file && !outputOptions[0].dir;
```

Comment:
Even though I just merged this, there is one minor thing I just noticed: If inputOptions is an array of length one and you specify neither `output.dir` nor `output.file`, generate will return `undefined` for the code and rollup will fail. I will revert this particular change for this release for now as the previous behaviour also better corresponded to this check https://github.com/rollup/rollup/blob/c7981c447677d06a3646cc26299eb5dc357998db/src/rollup/index.ts#L235 and we'll need to find a better solution here